service/hub/config/server.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Maybe we must do not commit keys, for develop tests, just create key pair and delete it when tests done, and for prod env, using configmap+secrets  or reading envs from command line will works fine.

Reply:
ok